.github/labeler.yml

```
@@ -0,0 +1,2 @@
+triage:
+- any: ['*']
```

Should we add EOL here?
```suggestion
- any: ['*']

```